src/Evolution/Systems/GeneralizedHarmonic/Constraints.cpp

```diff
@@ -100,10 +100,10 @@ void two_index_constraint_add_term_4_of_11(
 template <size_t SpatialDim, typename Frame, typename DataType>
 void two_index_constraint_add_term_5_of_11(
     const gsl::not_null<tnsr::ia<DataType, SpatialDim, Frame>*> constraint,
-    const tnsr::ia<DataType, SpatialDim, Frame>& d_gauge_function) noexcept {
+    const tnsr::ab<DataType, SpatialDim, Frame>& d_gauge_function) noexcept {
```

Comment:
because the spacetime deriv of gauge_function is already available in the databox, instead of extracting the spatial deriv elsewhere, we just now do that in here itself.